org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet144.d

```diff
@@ -109,7 +109,7 @@ void main() {
                 auto t = elapsed.stop() * 1_000;
             } else { // Phobos
                 elapsed.stop();
-                auto t = elapsed.peek.msecs;
+                auto t = elapsed.peek().total!"msecs"();
```

Comment:
Why this change?

Reply:
The peek() of deprecated `std.datetime.StopWatch` returns TickDuration, However peek() of `std.datetime.stopwatch.StopWatch` returns `Duration` (`TickDuration` is deprecated). To take a millisecond value from `Duration` we need to use the total().

Reply:
Ok, I see.